src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -86,7 +86,15 @@
     static {
         Properties prop = new Properties();
         try {
-            prop.load(new FileInputStream("src/test/resources/test.properties"));
+            String propertiesFilename;
+            if (isTravis()) {
+                propertiesFilename = "test.travis.properties";
+            } else if (isAppveyor()) {
+                propertiesFilename = "test.appveyor.properties";
+            } else {
+                propertiesFilename = "test.properties";
+            }
+            prop.load(new FileInputStream("src/test/resources/" + propertiesFilename));
```

Comment:
The only differing property is the browser. I don't think it's too much to modify the code to selectively choose the browser based on the environment. Everything else can be hardcoded as the default, just like how it was done before.